docs/src/pages/components/authenticator/hidesignup.default.vue.mdx

```diff
@@ -1,3 +1,10 @@
-```html{2} file=../../../../../examples/vue/src/pages/ui/components/authenticator/sign-in-with-email/index.vue#L10-
-
+```html{2}
+<template>
+  <authenticator :hide-sign-up="true">
+    <template v-slot="{ user, signOut }">
+      <h1>Hello {{ user.username }}!</h1>
+      <button @click="signOut">Sign Out</button>
```

Comment:
This is a bit of duplication, since it's similar to what's int he `sign-in-with-email/index.vue` file. But I think this is fine, for these small examples.